src/main/java/org/opensrp/repository/postgres/TaskRepositoryImpl.java

```diff
@@ -427,6 +427,11 @@ public Task updateTask(Task task) {
 		return convertToFHIRTasks(convert(taskMetadataMapper.selectMany(example, 0, DEFAULT_FETCH_SIZE)));
 	}
 
+	@Override
```

Comment:
Could we add some javadocs here to explain why null is returned

Reply:
I think we should make it similar to https://github.com/opensrp/opensrp-server-core/blob/81b0984ab1468f1f271167aa9ff83a80f15fc68d/src/main/java/org/opensrp/repository/postgres/TaskRepositoryImpl.java#L424.